modules/cstrike/cstrike/CstrikeNatives.cpp

```diff
@@ -2011,7 +2011,9 @@ static cell AMX_NATIVE_CALL cs_draw_progress_bar(AMX *amx, cell *params)
 	int index = params[1];
 
 	if(index)
+	{
 		CHECK_PLAYER(index);
+	}
```

Comment:
That's because `CHECK_PLAYER` macro isn't correctly written. Look at [RETURN_META macro](https://github.com/theAsmodai/metamod-r/blob/0fff97941f46ccb113d4525e36ef9d814b3ccba7/metamod/extra/example/include/metamod/meta_api.h#L59-L60), it doesn't have this problem.

Reply:
@WPMGPRoSToTeMa Yeah, I didn't have this problem when I compiled it on my computer as well.